install_config/revhistory_install_config.adoc

```diff
@@ -5,6 +5,22 @@
 :icons:
 :experimental:
 
+== Thu Feb 25 2016
+
+//tag::install_config_thu_feb_25_2016[]
+[cols="1,3",options="header"]
+|===
+
+|Affected Topic |Description of Change
+
+|link:../openshift-enterprise/enterprise/3.0//install_config/install/advanced_install.html[Installing ->
```

Comment:
@ahardin-rh Reminder for Monday's publish: this link is wonky. Also, `welcome/full_revhistory.adoc` didn't get updated for the Feb 25 update.